src/SmartComponents/ActivityTable/Columns.js

```diff
@@ -17,28 +23,140 @@ TaskNameCell.propTypes = {
+
+  popoverBodyContent = '';
+  hasPopover = true;
+  switch (task.status) {
+    case TASK_STATUS.RUNNING:
+      badgeColor = '#2B9AF3';
+      iconStatus = 'info';
+      IconType = InProgressIcon;
+      popoverBodyContent = [
+        `${task.running_jobs_count || 0} Running`,
+        `${task.completed_jobs_count || 0} Completed`,
+      ];
+      break;
+    case TASK_STATUS.COMPLETED:
+      badgeColor = '#3E8635';
+      iconStatus = 'success';
+      IconType = CheckCircleIcon;
+      hasPopover = false;
+      break;
+    case TASK_STATUS.COMPLETED_WITH_ERRORS:
+      badgeColor = '#FF0000';
+      iconStatus = 'danger';
+      IconType = ExclamationCircleIcon;
+      popoverBodyContent = [
+        `${task.failure_jobs_count || 0} Failed`,
+        `${task.timeout_jobs_count || 0} Timeout`,
+        `${task.running_jobs_count || 0} Running`,
+        `${task.completed_jobs_count || 0} Completed`,
+      ];
+      break;
+    case TASK_STATUS.FAILURE:
+      badgeColor = '#FF0000';
+      iconStatus = 'danger';
+      IconType = ExclamationCircleIcon;
+      popoverBodyContent =
+        'All jobs in this task failed to run on their target systems.';
+      break;
+    case TASK_STATUS.CANCELLED:
+      badgeColor = '#FF0000';
+      iconStatus = 'danger';
+      IconType = ExclamationCircleIcon;
+      popoverBodyContent = 'All jobs in this task have been cancelled';
+      break;
+  }
+
+  return { badgeColor, iconStatus, IconType, popoverBodyContent, hasPopover };
+};
+
+const formatPopoverBodyContent = (popoverBodyContent) => {
+  if (typeof popoverBodyContent === 'string') {
+    return <Text style={{ paddingLeft: '1rem' }}>{popoverBodyContent}</Text>;
+  }
+
+  let gridItems = [];
+  popoverBodyContent.forEach((item) => {
+    const [count, status] = item.split(' ');
+    const gridItem = (
+      <React.Fragment>
+        <GridItem span={2}>
+          <Text style={{ textAlign: 'right' }}>{count}</Text>
+        </GridItem>
+        <GridItem span={10}>
+          <Text style={{ textAlign: 'left' }}>{status}</Text>
+        </GridItem>
+      </React.Fragment>
+    );
+    gridItems.push(gridItem);
+  });
+  return (
+    <Grid hasGutter style={{ columnGap: '1rem', rowGap: '0' }}>
+      {gridItems}
+    </Grid>
+  );
+};
+
+const StatusCell = (task) => {
+  if (typeof task.status !== 'string') {
+    return;
+  }
+
+  const badgeStyle = {
+    display: 'flex',
+    alignItems: 'center',
+    marginRight: '4px',
+  };
+
+  const { badgeColor, iconStatus, IconType, popoverBodyContent, hasPopover } =
+    getStatusProps(task);
+
+  const coloredBadgeStyle = { ...badgeStyle, color: badgeColor };
+  const statusContent = (
+    <span>
+      <Icon status={iconStatus} style={{ marginRight: '4px' }}>
+        <IconType />
+      </Icon>
+      {!hasPopover ? (
+        <span>{task.status}</span>
+      ) : (
+        <span className="popoverLink">{task.status}</span>
```

Comment:
We should use this component to handle the underline: https://www.patternfly.org/components/description-list/#term-help-text